assets/popup.html

```diff
@@ -4,13 +4,10 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Home Assistant</title>
-    <link rel="stylesheet" href="popup.css" />
-    <script src="popup.js"></script>
+      <link rel="stylesheet" href="popup.css" />
   </head>
   <body>
-    <section class="is-hidden" id="alert">
-      <p class="title">The Home Assistant extension in not configured</p>
-      <p class="subtitle"><a href="/options.html" target="_blank">Open options</a></p>
-    </section>
+  <div id="popup"></div>
+  <script src="popup.js"></script>
   </body>
-</html>
+</html>
```

Comment:
Did you bypass the git hook?

Please run `npx prettier -w .` to format your code

Reply:
Ok, I've done it now, let me know when you finish review, so I'll commit the changes.